tools/clang/test/HLSLFileCheck/hlsl/workgraph/mesh_node_attribs_rdat.hlsl

```diff
@@ -217,6 +201,8 @@ void node02_mesh_maxdispatch(DispatchNodeInputRecord<RECORD> input) {}
 // CHECK:      }
 // CHECK:    }
 // CHECK:  }
+// Lots of intermediate RDAT output that is skipped,
+// we jump to checking for ID3D12Library Reflection here.
```

Comment:
This is ok, since these table elements are expanded in-line and checked inside the `RuntimeDataFunctionInfo*` record.

Reply:
Understandable, just thought I'd leave that comment there to prevent any confusion in the future.